edx_repo_tools/auth.py

```diff
@@ -158,13 +164,15 @@ def explode(hub, dry):
         help='Enable debug logging',
         default=False
     )
-    @click.pass_context
     @functools.wraps(f)
-    def wrapped(ctx, username, password, token, debug, *args, **kwargs):
+    def wrapped(username, password, token, debug, *args, **kwargs):
+
         if debug:
             logging.basicConfig()
             logging.getLogger().setLevel(logging.DEBUG)
 
         hub = login_github(username, password, token)
-        ctx.invoke(f, hub, *args, **kwargs)
+
+        f(hub=hub, *args, **kwargs)
```

Comment:
And it turns out I was wrong. (And using invoke made it impossible to stack decorators).